packages/core/src/CorePlugin.ts

```diff
@@ -1,7 +1,7 @@
 import { JWPlugin } from './JWPlugin';
 import JWEditor from './JWEditor';
 import { VRangeDescription } from './VRange';
-import { Direction, RelativePosition } from '../../utils/src/range';
+import { RelativePosition, Direction } from '../../utils/src/range';
```

Comment:
@dmo-odoo This is your commit. Could you please write a message and put yourself as author in it? Thanks.